Formula/cannoli.rb

```diff
@@ -0,0 +1,23 @@
+class Cannoli < Formula
+  desc "Big Data Genomics ADAM Pipe API wrappers for bioinformatics tools"
+  homepage "https://github.com/bigdatagenomics/cannoli"
+  url "https://search.maven.org/remotecontent?filepath=org/bdgenomics/cannoli/cannoli-distribution-spark2_2.11/0.2.0/cannoli-distribution-spark2_2.11-0.2.0-bin.tar.gz"
+  sha256 "919f695ac91e7e7e9ccd5e008b3300c3eb28581cdf9bebaa713935059aeaa1ca"
+
+  head do
+    url "https://github.com/bigdatagenomics/cannoli.git", :shallow => false
+    depends_on "maven" => :build
+  end
+
+  depends_on "apache-spark"
+
+  def install
+    system "mvn", "clean", "install", "-DskipTests=true" if build.head?
+    libexec.install Dir["*"]
+    bin.write_exec_script Dir["#{libexec}/bin/*"]
```

Comment:
In most situations you can use `install_symlink` rather than `write_exec_script`. Is that true here?
```ruby
bin.install_symlink Dir["#{libexec}/bin/*"]
```

Reply:
I tried this and it worked for me locally.